### ISSUE TITLE
Starting FreeBSD 10.3 (or younger) DHCP VNET jail doesn't correctly display IP/Mask at startup

### ISSUE THREAD

**Issue by rexpo** [iocage/iocage#514]
This problem was introduced in 6556e18 due to its use of the `ifconfig -f inet:cidr` command, which didn't exist prior to FreeBSD 11.

- [x] Supply `iocage --version`

Version	1.0 ALPHA
0271b76

- [x] Supply the commands used, along with any steps to recreate it.

```
iocage create --name oc-template --release 10.3-RELEASE vnet=on bpf=yes dhcp=on interfaces="vnet0:bridge20"
iocage start oc-template
```

- [x] Provide the output from the command you issued.

```
* Starting oc-template
  + Started OK
  + Configuring VNET OK
ifconfig: illegal option -- f
usage: ifconfig [-L] [-C] [-g groupname] interface address_family [address [dest_address]]
                [parameters]
       ifconfig interface create
       ifconfig -a [-L] [-C] [-g groupname] [-d] [-m] [-u] [-v] [address_family]
       ifconfig -l [-d] [-u] [address_family]
       ifconfig [-L] [-C] [-g groupname] [-d] [-m] [-u] [-v]
  + DHCP Address: ERROR, check jail logs
  + Starting services OK
```

- [x] Supply what you expected the result or output to be

```
* Starting oc-template
  + Started OK
  + Configuring VNET OK
  + DHCP Address: 10.1.1.234/26
  + Starting services OK
```

- [x] Checked that the problem has not already been fixed on `master` if using
 a stable release.

**Comment by rexpo** [comment]
Sorry I don't have a 10.4-RELEASE machine handy at the moment, haven't had a chance to test anywhere else either :/

Anyway, no worries, my local copy is working nicely on 10.3-RELEASE ;)

**Comment by rexpo** [comment]
I finally got around to testing this with 10.4-RELEASE - same problem.

**Comment by rexpo** [comment]
This has been fixed in e965a98 but I have only tested on 10.3-RELEASE and 10.4-RELEASE so far. (Should be fine on newer releases, as we're using "the good old way" of extracting IP & CIDR)

What's the standard practice at this point, @skarekrow? Do we wait for testing on >= 11.0-RELEASE? If not, should I just close this issue myself, or are you the boss of closing issues?

**Comment by skarekrow** [comment]
We'll close for now and if somebody has an issue they can reopen or file a new bug :)
